packages/site/src/layout/AnimatedPresenceDisclosure.tsx

```diff
@@ -14,27 +14,40 @@ function AnimatedPresenceDisclosure({
   to,
 }: AnimatedPresenceDisclosureProps) {
   const [isOpen, setIsOpen] = useState(false);
+  const [isHovered, setIsHovered] = useState(false);
 
   const handleButtonClick = (e: React.MouseEvent) => {
     e.preventDefault();
     setIsOpen(!isOpen);
   };
 
+  const handleHover = (e: React.MouseEvent) => {
+    e.preventDefault();
+    setIsHovered(!isHovered);
+  };
+
   return (
-    <>
+    <div>
       <Link
         to={to ?? "/"}
         href="https://www.google.com"
         tabIndex={0}
         onFocus={e => (e.currentTarget.style.boxShadow = "0 0 3px #0073e6")}
```

Comment:
Can we use the `--shadow-focus` token here?

Also noticing that the collapsable headers have an unstyled focus state:
![image](https://github.com/user-attachments/assets/c11bfa3c-ffa0-46d4-a63c-30186d58adcc)

Reply:
Focus states are addressed in https://jobber.atlassian.net/browse/JOB-111070